airflow/configuration.py

```diff
@@ -580,10 +589,24 @@ def getint(self, section, key):
     def getfloat(self, section, key):
         return float(self.get(section, key))
 
+    def remove_option(self, section, option):
+        try:
+            ConfigParser.remove_option(self, section, option)
+        except ConfigParser.NoSectionError:
+            pass
+        try:
+            ConfigParser.remove_option(self.defaults, section, option)
+        except ConfigParser.NoSectionError:
+            pass
+
     def read(self, filenames):
         ConfigParser.read(self, filenames)
         self._validate()
 
+    def has_option(self, section, option):
```

Comment:
I think it would be better to return `True` for all possible options including (env, cmd, defaults, ...).

I think that not overriding `has_option` could cause potential problem. Let's assume that the given situation.
In somewhere of the code, I want to get a config from the the `AirflowConfigParser` as a developer. I would check there is the config before I get it. However it could say it doesn't have the config even tough it is in `default` config.

Would it be wired?